src/daft-cli/src/python.rs

```diff
@@ -51,14 +51,14 @@ fn run_dashboard(py: Python, args: DashboardArgs) {
     let socket_addr = SocketAddr::from((args.addr, args.port));
 
     // Set the subscriber for the detached run
-    tracing_subscriber::registry()
+    let _ = tracing_subscriber::registry()
         .with(
             tracing_subscriber::EnvFilter::builder()
                 .with_default_directive(filter)
                 .from_env_lossy(),
         )
         .with(tracing_subscriber::fmt::layer())
-        .init();
+        .try_init();
```

Comment:
The error seems to be some reinitialization error between our existing pyo3-log and tracing which sets up tracing-log. I couldn't disable tracing-log so I just ignore the error. It doesn't actually break any tracing stuff